.github/workflows/main.yml

```diff
@@ -15,21 +15,27 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.7
+    - uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
     - name: Install requirements
       run: |
         pip install -r requirements.txt
         pip install pytest pytest-cov flake8 black mypy
-    - name: Run tests
```

Comment:
Why remove tests?  I don't know much about the GitHub actions as Mark set it up.

Reply:
It was just moved below. I curious to know why you moved it, though.

Reply:
Because linting is fast and tests are slow. I don't want to wait for slow tests just to have the 5s linting test fail afterwards.